### PR TITLE
[HEIDELPAYSUPPORT-108] Validate birthdate for unzer installment

### PR DESCRIPTION
**Pull request by JoeKre**

### CODE REVIEW

File: src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
             "deviceDeleted": "Das Zahlungsmittel wurde erfolgreich entfernt.",
             "title": "Ihre gespeicherten Zahlungsmittel"
         },
-        "finishPaymentFailed": "<a href=\"%editOrderUrl%\">Zahlung ändern</a>"
+        "finishPaymentFailed": "<a href=\"%editOrderUrl%\">Zahlung ändern</a>",
+        "birthdayInvalid": "Geburtsdatum ungültig"
```
Comment (sixer1182) [inline src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json:67]: What about "You need to be at least 18 years of age".

Comment (sobyte) [reply src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json:67]: Do you only want to change the general error message or do we need to implement multiple cases for displaying different error messages?

Comment (sixer1182) [reply src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json:67]: Please do two distinct checks and error messages.
1. Invalid date (from format, invalid date e.g. 30.2.2001)
2. Buyer is too young (younger than 18 years)

Comment (sobyte) [reply src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json]: @sixer1182 As discussed: We've just changed the text to the second message.

File: src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json [hunk 2]
```diff
@@ -64,6 +64,6 @@
             "title": "Ihre gespeicherten Zahlungsmittel"
         },
         "finishPaymentFailed": "<a href=\"%editOrderUrl%\">Zahlung ändern</a>",
-        "birthdayInvalid": "Geburtsdatum ungültig"
+        "birthdayInvalid": "Du musst mindestens 18 Jahre alt sein."
```
Comment (sixer1182) [inline src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json:67]: Are messages in Shopware informal in general ('Du' instead of 'Sie')?

Comment (JoeKre) [reply src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json]: Updated the message to a formal version based on other messages in the plugin 'Wählen Sie...', 'Ihre gespeicherten Zahlungsmittel'

**Review by sixer1182 — APPROVED** [review]

**Pull request merged**